CMakeLists.txt

```diff
@@ -200,11 +200,14 @@ add_custom_target(check COMMAND ctest --output-on-failure -C $<CONFIG> DEPENDS t
 add_custom_target(tests DEPENDS hilti-tests spicy-tests)
 
 # Packaging.
-# Check tags the HEAD commit corresponds to.
+# Check tags the HEAD commit corresponds to. If we cannot get the current
+# commit (e.g., no .git directory present for release tarballs), this will
+# leave SPICY_COMMIT unset which is fine for users working from tarballs.
 execute_process(COMMAND git tag --points-at
-    OUTPUT_VARIABLE GIT_TAGS)
+    OUTPUT_VARIABLE SPICY_TAGS
+    ERROR_VARIABLE ignored)
 
-if ( "${GIT_TAGS}" STREQUAL "" )
+if ( "${SPICY_TAGS}" STREQUAL "" )
```

Comment:
Independent of this patch, but I'm wondering if we should change this if-condition to check only for version tags  (i.e., matching `v*`) to avoid some unrelated tag unintentionally triggering the `else` branch?